contract/RadicleCloudBase.sol

```diff
@@ -0,0 +1,191 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity >=0.7.0 <0.9.0;
+
+/// @title RadicleCloudBase does on-chain accounting and emits events for operator.
+/// @dev RadicleCloudBase is an abstract contract.
+abstract contract RadicleCloudBase {
+    /// @dev Owner is needed to potentially re-adjust prices in future and withdraw funds.
+    address private owner;
+    /// @notice Keep track of expiry blocks for Orgs
+    mapping(address => uint64) public expiry;
+    /// @notice Remember owner of Org, one who initially created it
+    mapping(address => address) public orgOwner;
```

Comment:
It's confusing to use the term "org" in the code/comments, they should be "subscriptions" or "deployments".

Reply:
I still had to use term `org` at some places for clarity but changed methods as much as possible to `deployment`. Take another look please.